global-script.lua

```diff
@@ -1989,7 +1989,23 @@ function handlePiece(object, depth)
     elseif object.getName() == "Blight" then
         object = resetPiece(object, Vector(0,180,0), depth)
     elseif string.sub(object.getName(),-7) == "Defence" then
-        if object.getLock() == false then object.destruct() end
+        -- Reset defense tokens to highest locked state.
+        -- Lock status is linked tied to a specific state.
+        local state = 21
```

Comment:
instead of a hardcoded magic number, you should use the length of `getStates()` + 1